content/testing/fluent-assertions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: FluentAssertions
+showMetadata: true
+editable: true
+showToc: true
+---
+
+สมมติว่าเรามี Data Structure ดังนี้ คำถามคือเราจะ Assert อย่างไร
+
+```c#
+public class Todo
+{
+    public int id  { get; set; }
+    public string title  { get; set; }
+    public bool completed  { get; set; }
+}
+
+public class TodoList
+{
+
+    public IEnumerable<Todo> GetTodoList()
+    {
+        return new Todo[]
+        {
+            new Todo{
+                id = 0,
+                title = "Shopping",
+                completed = false
+            },
+            new Todo{
+                id = 1,
+                title = "Running",
+                completed = true
+            },
+            new Todo{
+                id = 2,
+                title = "Reading a book",
+                completed = false
+            }
+        };
+    }
+}
+```
+
+ถ้าใช้ xunit อย่างเดียว อาจจะยุ่งยาก ลองใช้ [FluentAssertions](https://www.nuget.org/packages/FluentAssertions)
+
+# Getting Started
+
+1. ติดตั้ง [FluentAssertions](https://www.nuget.org/packages/FluentAssertions)
+
+    ```sh
+    dotnet add package FluentAssertions --version 5.10.3
```

Comment:
Very minor.
I think you can omit a version and  if is the first time nuget will install the latest release version and after that it will install a minor version if you restore packages on a new computer.

Reply:
Good comment this code base on version 5 , it may be wrong when people read this docs when some func is deprecated. 

i use the same way of ms Docs